test/serializer/basic/NestedFunctions4.js

```diff
@@ -0,0 +1,16 @@
+// Copies of x0: 3
+f = function() {
+  function nested() {
+    var x0 = 1;
+    var x1 = x0 + x0;
+    var x2 = x1 + x1;
+    var x3 = x2 + x2;
+    var x4 = x3 + x3;
+    return x4;
+  }
+  return nested;
+}
+g = f();
+inspect = function() {
```

Comment:
The test runner doesn't actually execute tests with a `// Copies of` comment, it just checks that the code abides by that search. 

Request Changes: Please add a test case that runs the inspect function (or change the test runner)

Reply:
Hmm, I do not think that is a problem of this PR: all the tests using "Copies of" has "inspect" function in it. That test runner bug is a separate issue which I do not think it should hold this PR.
Note: checking "Copies of" is the major purpose of this test.